scrapy/core/downloader/handlers/__init__.py

```diff
@@ -40,8 +47,12 @@ def _get_handler(self, scheme):
             return None
 
         path = self._schemes[scheme]
+        dhcls = load_object(path)
+        self._load_handler(scheme, dhcls)
+        return self._handlers[scheme]
+
+    def _load_handler(self, scheme, dhcls):
```

Comment:
could you please check the logging.error call below? `_load_handler` method doesn't have `path` as an argument, but uses it in a message. Probably the idea was to catch load_object errors as well?

Reply:
Indeed, trying to rush things at the end of the day on Friday is never a good thing.